rocAL/rocAL/CMakeLists.txt

```diff
@@ -173,18 +185,23 @@ if(${BUILD_RALI})
     else()
         target_compile_definitions(${PROJECT_NAME} PUBLIC ENABLE_OPENCV=0)
     endif()
-    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -O3 -fopenmp -msse4.2 -mavx2 -Wall -ljsoncpp -fPIC -pg -pthread -std=c++14")
+    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -O3 -fopenmp -msse4.2 -mavx2 -Wall -ljsoncpp -fPIC -pg -pthread -std=c++14 -Wno-deprecated-declarations")
     add_definitions(-ljsoncpp)
     message("-- ${Green}rocAL - CMAKE_CXX_FLAGS:${CMAKE_CXX_FLAGS}")
 
     if(NOT FFMPEG_FOUND)
         message("-- ${Yellow}rocAL library is going to be built without video decode functionality ${ColourReset}")
         target_link_libraries(${PROJECT_NAME} -fPIC ${PROTOBUF_LIBRARIES} lmdb boost_system boost_filesystem turbojpeg openvx vx_rpp jsoncpp)
     else()
-        message("-- ${Green}rocAL library is going to be built with video decode functionality ${ColourReset}")
-        target_link_libraries(${PROJECT_NAME} -DRALI_VIDEO -fPIC ${PROTOBUF_LIBRARIES} lmdb turbojpeg openvx vx_rpp  jsoncpp)
+        if(GPU_SUPPORT AND "${BACKEND}" STREQUAL "OPENCL")  # rocAL Video module yet to be supported with HIP BACKEND
```

Comment:
do we support video for HIP backend?

Reply:
No, currently video support has not been added for the HIP Backend